data_collection/gazette/spiders/to/to_cariri_do_tocantins.py

```diff
@@ -0,0 +1,16 @@
+import datetime as dt
+
+from gazette.spiders.base.nucleogov import NucleoGovGazetteSpider
+
+
+class ToCaririDoTocantinsSpider(NucleoGovGazetteSpider):
+    name = "to_cariri_do_tocantins"
+    TERRITORY_ID = "1703867"
+    allowed_domains = [
+        "dom.cariri.to.gov.br",
+    ]
+    url_base = (
+        "https://dom.cariri.to.gov.br/api/diarios?data={}&calendar=true&situacao=2"
+    )
```

Comment:
Reflete aqui a mudança na classe base

```suggestion
    url_base = "https://dom.cariri.to.gov.br"
```